packages/manager/modules/sms/src/sms/batches/create/telecom-sms-batches-create.html

```diff
@@ -0,0 +1,99 @@
+<header>
+    <oui-back-button data-on-click="$ctrl.goBack()">
+        <h1 data-translate="sms_batches_create_title"></h1>
+    </oui-back-button>
+</header>
+
+<section class="container-fluid mt-4 ml-4">
+    <div class="row">
+        <div class="col-md-9">
+            <oui-message
+                type="error"
+                data-ng-if="$ctrl.error != null"
+                dismissable
+                on-dismiss="$ctrl.error = null"
+            >
+                <span
+                    data-translate="sms_batches_create_error"
+                    data-translate-values="{ 'error': $ctrl.error.data.message }"
+                ></span>
+            </oui-message>
+            <form name="$ctrl.configuration" novalidate>
+                <ovh-manager-sms-sms-configuration
+                    go-to-phonebooks-contacts="$ctrl.goToPhonebooksContacts"
```

Comment:
data- for all the custom attributes

Reply:
As you said, it is custom attributes, so it isn't necessary to add the prefix